evms/parity.go

```diff
@@ -133,6 +131,8 @@ func (evm *ParityVM) Copy(out io.Writer, input io.Reader) {
 						stateRoot.StateRoot = string(root[:66])
 					}
 				}
+				// Try to unmarshal the state root
+				json.Unmarshal(data, &stateRoot)
```

Comment:
Made it a bit more clear. Basically currently we can only unmarshal the state root if an error occurred.
This will make it possible to also grab the state root if no error occurred.